rfd/0079/jenkins-mariposa.md

```diff
@@ -0,0 +1,76 @@
+# jupiter.example.com, Jenkins, and Mariposa
```

Comment:
Can you put this in [`00036/user-stories/jupiter-example-come-jenkins.md`](https://github.com/misterbisson/rfd/tree/master/rfd/0036/user-stories) and add it to the user stories in [the main readme](https://github.com/misterbisson/rfd/blob/master/rfd/0036/README.md)?